### PR TITLE
Onboarding gas transfer

### PR DESCRIPTION
**Pull request by lpgeiger**
- [x] user who sends an onboarding transaction will send gas to owner and operator keys of the new validator
- [x] patch tests for balances on both sides.

### CODE REVIEW

File: language/stdlib/modules/0L/LibraAccount.move [hunk 1]
```diff
@@ -942,20 +940,17 @@ module LibraAccount {
...
         payee: address
     ) acquires LibraAccount, Balance, AccountOperationsCapability {
-        let bootstrap_amount = 1000000; // 1 gascoin, without scaling representation 
         let payer_addr = Signer::address_of(payer_sig);
         let account_balance = borrow_global_mut<Balance<Token>>(payer_addr);
         let balance_coin = &mut account_balance.coin;
-
-        assert(Libra::value(balance_coin) > bootstrap_amount * 2, Errors::limit_exceeded(EINSUFFICIENT_BALANCE));
+        assert(Libra::value(balance_coin) > BOOTSTRAP_COIN_VALUE, Errors::limit_exceeded(EINSUFFICIENT_BALANCE));
         // Should abort if the
```
Comment (nelaturuk) [inline language/stdlib/modules/0L/LibraAccount.move:951]: Can we assert that payee is a validator here?

Comment (nelaturuk) [reply language/stdlib/modules/0L/LibraAccount.move:951]: Also, the balance_coin should not become 0 after they transfer the value. Can we ensure it is 2x?

File: language/stdlib/modules/0L/LibraAccount.move [hunk 2]
```diff
@@ -942,10 +942,14 @@ module LibraAccount {
         payer_sig: &signer,
         payee: address
     ) acquires LibraAccount, Balance, AccountOperationsCapability {
+        let bootstrap_amount = 1000000; // 1 gascoin, without scaling representation
```
Comment (nelaturuk) [inline language/stdlib/modules/0L/LibraAccount.move]: Cant we have this as a const at the top?

File: language/stdlib/modules/0L/LibraAccount.move (hunk 1 above)
Comment (lpgeiger) [reply language/stdlib/modules/0L/LibraAccount.move:951]: > Can we assert that payee is a validator here?
Yes probably, good idea.

> Also, the balance_coin should not become 0 after they transfer the value. Can we ensure it is 2x?
Yes I think there are multiple checks in there for that.

File: language/stdlib/modules/0L/LibraAccount.move (hunk 2 above)
Comment (lpgeiger) [reply language/stdlib/modules/0L/LibraAccount.move]: Yes good idea

**Pull request merged**